wire-ios/Wire-iOS/Sources/UserInterface/Settings/DeviceManagement/ClientTableViewCell.swift

```diff
@@ -75,19 +56,13 @@ class ClientTableViewCell: UITableViewCell, DynamicTypeCapable {
             } else if userClient.isLegalHoldDevice {
                 nameLabel.text = L10n.Localizable.Device.Class.legalhold
             }
-
+            let proteusId = userClient.displayIdentifier.fingerprintStringWithSpaces.uppercased()
```

Comment:
suggestion: Maybe here we need to use .localizedUppercased to make sure the end result is localized.